pkg/storaged/client.js

```diff
@@ -443,24 +443,22 @@ function init_model(callback) {
 
     function enable_udisks_features() {
         if (!client.manager.valid)
-            return cockpit.resolve();
+            return Promise.resolve();
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18487-20230310-143650-f841503b-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)